JuiceMaker/JuiceMaker/Controller/ViewController.swift

```diff
@@ -7,12 +7,71 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    var juiceMaker = JuiceMaker()
+    
+    @IBOutlet weak var strawberryStock: UILabel!
```

Comment:
```suggestion
    @IBOutlet weak var strawberryStockLabel: UILabel!
```
끝에 Label이라고 명시를 해야 좀더 변수가 분명해보이네요 😄

Reply:
Label을 추가해 수정했습니다!